tools/ete/homology_classifier.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import print_function
+
+import optparse
+
+from ete3 import PhyloTree
+
+
+def main():
+    usage = "usage: %prog --genetree <genetree-file> --speciestree <speciestree-file> [options]"
+    parser = optparse.OptionParser(usage=usage)
+    parser.add_option('--genetree', help='GeneTree in nhx format')
+    parser.add_option('--out_format', type='string', default='tabular', help='Choose output format')
+    parser.add_option('--filters', default=[], help='Filter families')
+
+    options, args = parser.parse_args()
+
+    if options.genetree is None:
+        parser.error("--genetree option must be specified, GeneTree in nhx format")
+
+    # reads single gene tree
+    genetree = PhyloTree(options.genetree)
+
+    leaves_list = genetree.get_leaf_names()
+
+    species_list = []
+    gene_lose_list = []
+
+    for i, val in enumerate(leaves_list):
+        if '_' in leaves_list[i]:
+            species_list.append(leaves_list[i].split("_")[1])
+        else:
+            gene_lose_list.append(i)
+
+    leaves_list = [i for j, i in enumerate(leaves_list) if j not in gene_lose_list]
+
+    species_dict = {}
+
+    for val in species_list:
+        count = "one"
+        if val in species_dict:
+            count = "many"
+        species_dict[val] = count
+
+    homologis = {
+        'one-to-one': [],
+        'one-to-many': [],
+        'many-to-one': [],
+        'many-to-many': [],
+        'paralogs': []
+    }
+
+    # stores relevant homolgy types in dict
+    for i, val in enumerate(leaves_list):
+        for j in range(i + 1, len(leaves_list)):
```

Comment:
`for leaf2 in leaves_list[i + 1, ]:`

Reply:
This throws error "TypeError: list indices must be integers or slices, not tuple"